api/nakadi-event-bus-api.yaml

```diff
@@ -754,6 +790,10 @@ paths:
           description: Client is not authenticated
           schema:
             $ref: '#/definitions/Problem'
+        '403':
+          description: Access forbidden because of EventType authorization failure.
```

Comment:
is this the only reason for this response code? Or can this code also be returned for callers without the required oauth scope? if the latter, it might be worth including that possibility in the description.